setup.py

```diff
@@ -130,7 +130,7 @@ def setup_package():
         install_requires=install_reqs,
         setup_requires=['flake8'],
         cmdclass=cmdclass,
-        tests_require=['pytest-cov', 'pytest'],
+        tests_require=['pytest-cov', 'pytest', 'mock'],
```

Comment:
See the comment to test_configuration. Without this requirement the tests fail in python 3.4.